packages/data/src/world/mq/jabu_jabu_mq.yml

```diff
@@ -84,22 +200,43 @@
     "MQ Jabu-Jabu Wonder Item Falling Like-Likes Explosion 1": "has_explosives"
     "MQ Jabu-Jabu Wonder Item Falling Like-Likes Explosion 2": "has_explosives"
     "MQ Jabu-Jabu Wonder Item Falling Like-Likes Explosion 3": "has_explosives"
+"Jabu-Jabu Back Forked Paths Right 2":
+  dungeon: JJ
+  exits:
+    "Jabu-Jabu Back Forked Paths": "true" #One-sided collision on the web
+  events:
+    JABU_TENTACLE_BLUE: "can_boomerang && soul_enemy(SOUL_ENEMY_PARASITE)"
+"Jabu-Jabu Back Forked Paths Left 1":
+  dungeon: JJ
+  exits:
+    "Jabu-Jabu Back Forked Paths": "true"
+  events:
+    JABU_TENTACLE_GREEN: "can_boomerang && soul_enemy(SOUL_ENEMY_PARASITE)"
+"Jabu-Jabu Back Forked Paths Left 2":
+  dungeon: JJ
+  exits:
+    "Jabu-Jabu Back Forked Paths": "true"
+  locations:
+    "MQ Jabu-Jabu GS Back": "gs && ((climb_anywhere && (can_hammer || can_use_mask_blast)) || (has_explosives && can_collect_distance))"
 "Jabu-Jabu Basement Side Room":
   dungeon: JJ
   exits:
-    "Jabu-Jabu Main": "true"
+    "Jabu-Jabu Ruto Section Bottom Floor": "event(JABU_MQ_BASEMENT_SIDE_BUTTON)" #Hard to tell if the Red Tentacle will interfere exiting the room with how close to the door it is.
+  events:
+    JABU_MQ_BASEMENT_SIDE_PLATFORM: "has_lens && soul_keese && soul_enemy(SOUL_ENEMY_STINGER) && has_ranged_weapon"
+    JABU_MQ_BASEMENT_SIDE_BUTTON: "true" #Listed here for action shuffle
   locations:
-    "MQ Jabu-Jabu GS Basement Side Room": "gs && ((has_lens && soul_keese && soul_enemy(SOUL_ENEMY_STINGER)) || (has_hover_boots && can_collect_distance) || (has_fire_arrows && can_longshot))"
-"Jabu-Jabu Pre-Boss":
+    "MQ Jabu-Jabu GS Basement Side Room": "gs && ((event(JABU_MQ_BASEMENT_SIDE_PLATFORM) && can_collect_distance) || (has_hover_boots && can_collect_distance) || (has_fire_arrows && can_longshot) || (climb_anywhere && can_collect_distance))"
```

Comment:
climb anywhere should be able to use any means that can destroy the GS, not just rang or hook.

Reply:
Changed.